pkgs/servers/home-assistant/default.nix

```diff
@@ -264,6 +264,7 @@ in with py.pkgs; buildPythonApplication rec {
     "folder"
     "folder_watcher"
     "freebox"
+    "fritz"
```

Comment:
> ERROR: Tests for component 'fritz' were enabled, but they do not exist!

Testes were added with v2021.5.

Reply:
See my comment at the very top.

Reply:
At the very bottom of the very top! :wink: